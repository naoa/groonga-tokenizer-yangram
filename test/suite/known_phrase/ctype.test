register tokenizers/yangram

table_create known_phrases TABLE_PAT_KEY ShortText
load --table known_phrases
[
{"_key": "【請求項4】"}
]
tokenize TokenYaBigram "【請求項４】検索クエリAが" NormalizerAuto --mode ADD
tokenize TokenYaBigram "【請求項４】検索クエリAが" NormalizerAuto --mode GET
