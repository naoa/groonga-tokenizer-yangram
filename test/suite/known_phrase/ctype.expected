register tokenizers/yangram
[[0,0.0,0.0],true]
table_create known_phrases TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
load --table known_phrases
[
{"_key": "【請求項4】"}
]
[[0,0.0,0.0],1]
tokenize TokenYaBigram "【請求項４】検索クエリAが" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "【請求項4】",
      "position": 0
    },
    {
      "value": "検索",
      "position": 1
    },
    {
      "value": "索ク",
      "position": 2
    },
    {
      "value": "クエ",
      "position": 3
    },
    {
      "value": "エリ",
      "position": 4
    },
    {
      "value": "リ",
      "position": 5
    },
    {
      "value": "a",
      "position": 6
    },
    {
      "value": "が",
      "position": 7
    }
  ]
]
tokenize TokenYaBigram "【請求項４】検索クエリAが" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "【請求項4】",
      "position": 0
    },
    {
      "value": "検索",
      "position": 1
    },
    {
      "value": "クエ",
      "position": 3
    },
    {
      "value": "エリ",
      "position": 4
    },
    {
      "value": "a",
      "position": 6
    },
    {
      "value": "が",
      "position": 7
    }
  ]
]
